### PR TITLE
l10n: uk: update translation

### PR DESCRIPTION
**Pull request by kasya**

### CODE REVIEW

File: po/uk.po [hunk 1]
```diff
@@ -20338,27 +20338,27 @@ msgstr "неприпустимий визначник фільтра \"%s\""
 #: list-objects-filter-options.c:128
 #, c-format
 msgid "must escape char in sub-filter-spec: '%c'"
-msgstr ""
+msgstr "необхідно екранувати символ у sub-filter-spec: \"%c\""
 
 #: list-objects-filter-options.c:171
 msgid "expected something after combine:"
-msgstr ""
+msgstr "щочь очікувалось після комбінування:"
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
msgstr "щоcь очікувалось після комбінування:"
```

File: po/uk.po [hunk 2]
```diff
@@ -20373,12 +20373,12 @@ msgstr "не вдалося розібрати дані часткового ф
 #: list-objects.c:116
 #, c-format
 msgid "entry '%s' in tree %s has tree mode, but is not a tree"
-msgstr ""
+msgstr "запис \"%s\" у дереві %s має режим дерева, але не є деревом"
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
msgstr "запис \"%s\" у дереві %s має режим tree, але не є деревом"
```

File: po/uk.po [hunk 3]
```diff
@@ -20405,40 +20414,40 @@ msgstr "Не вдалося створити \"%s.lock\": %s"
...
 #, c-format
 msgid "Failed to merge submodule %s (commits don't follow merge-base)"
-msgstr ""
+msgstr "Не вдалося злити підмодуль %s (коміти не відповідають базі)"
 
 #: merge-ort.c:1828 merge-ort.c:1837
 #, c-format
 msgid "Note: Fast-forwarding submodule %s to %s"
-msgstr ""
+msgstr "Примітка: Перемотування вперед підмодуля %s до %s"
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
msgstr "Примітка: перемотування підмодуля %s вперед до %s"
```

File: po/uk.po [hunk 4]
```diff
@@ -20470,21 +20483,27 @@ msgstr "Не вдалося додати %s до бази даних"
...
 
 #: merge-ort.c:2275 merge-recursive.c:2144
 #, c-format
 msgid ""
 "CONFLICT (implicit dir rename): Existing file/dir at %s in the way of "
 "implicit directory rename(s) putting the following path(s) there: %s."
 msgstr ""
+"КОНФЛІКТ (неявне перейменування каталогу): Існуючий файл/директорія за "
+"адресою %s заважає неявному перейменуванню директорії додавання наступних "
+"шляхів до: %s."
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
"КОНФЛІКТ (неявне перейменування каталогу): існуючий файл/директорія в "
"%s заважає неявному перейменуванню директорії, додавши наступні "
"шляхи до: %s."
```

File: po/uk.po [hunk 5]
```diff
@@ -20470,21 +20483,27 @@ msgstr "Не вдалося додати %s до бази даних"
...
 
 #: merge-ort.c:2287 merge-recursive.c:2154
 #, c-format
 msgid ""
 "CONFLICT (implicit dir rename): Cannot map more than one path to %s; "
 "implicit directory renames tried to put these paths there: %s"
 msgstr ""
+"КОНФЛІКТ (неявне перейменування директорії): Неможливо зіставити більше "
+"одного шляху з %s; неявне перейменування каталогів намагалося помістити туди"
+"ці шляхи: %s"
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
"КОНФЛІКТ (неявне перейменування директорії): Неможливо зіставити більше "
"одного шляху з %s; неявне перейменування директорій намагалося помістити туди"
"ці шляхи: %s"
```

File: po/uk.po [hunk 6]
```diff
@@ -20493,46 +20512,60 @@ msgid ""
 "renamed to multiple other directories, with no destination getting a "
 "majority of the files."
 msgstr ""
+"КОНФЛІКТ (розділення під час перейменування директорії): Неясно, куди "
+"перейменувати директорію %s; її було перейменовано до кількох інших "
+"директорій, при цьому більшість файлів не потрапило до жодної з них."
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
"КОНФЛІКТ (розбіжність під час перейменування директорії): Неясно, як "
"перейменувати директорію %s; її було перейменовано в декілька інших "
"директорій, жодна з яких не отримала більшості файлів."
```

File: po/uk.po [hunk 7]
```diff
@@ -20493,46 +20512,60 @@ msgid ""
...
 
 #: merge-ort.c:2528 merge-recursive.c:2490
 #, c-format
 msgid ""
 "WARNING: Avoiding applying %s -> %s rename to %s, because %s itself was "
 "renamed."
 msgstr ""
+"ПОПЕРЕДЖЕННЯ: Не застосовувати %s -> %s перейменування на %s, оскільки сам "
+"%s було перейменовано."
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
"ПОПЕРЕДЖЕННЯ: уникнення застосування %s -> %s перейменування на %s, оскільки "
"%s було перейменовано."
```

File: po/uk.po [hunk 8]
```diff
@@ -20493,46 +20512,60 @@ msgid ""
...
 
 #: merge-ort.c:2702 merge-recursive.c:3273
 #, c-format
 msgid ""
 "Path updated: %s added in %s inside a directory that was renamed in %s; "
 "moving it to %s."
 msgstr ""
+"Шлях було оновлено: %s додано до %s всередині директорії, яку було "
+"перейменовано у %s; переміщення до %s."
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
"перейменовано в %s; переміщення до %s."
```

File: po/uk.po [hunk 9]
```diff
@@ -20493,46 +20512,60 @@ msgid ""
...
 
 #: merge-ort.c:2710 merge-recursive.c:3280
 #, c-format
 msgid ""
 "Path updated: %s renamed to %s in %s, inside a directory that was renamed in "
 "%s; moving it to %s."
 msgstr ""
+"Шлях було оновлено: %s перейменовано на %s у %s, всередині директорії, яку "
+"було перейменовано у %s; переміщення до %s."
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
"Шлях було оновлено: %s перейменовано на %s в %s, всередині директорії, яку "
"було перейменовано в %s; переміщення до %s."
```

File: po/uk.po [hunk 10]
```diff
@@ -20493,46 +20512,60 @@ msgid ""
...
 
 #: merge-ort.c:2724 merge-recursive.c:3276
 #, c-format
 msgid ""
 "CONFLICT (file location): %s added in %s inside a directory that was renamed "
 "in %s, suggesting it should perhaps be moved to %s."
 msgstr ""
+"КОНФЛІКТ (розташування файлу): %s додано у %s всередині директорії, яку було"
+"перейменовано у %s, що свідчить про те, що його слід перемістити до %s."
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
"КОНФЛІКТ (розташування файлу): %s додано до %s всередині директорії, яку було"
"перейменовано в %s, що свідчить про те, що його слід перемістити до %s."
```

File: po/uk.po [hunk 11]
```diff
@@ -20493,46 +20512,60 @@ msgid ""
...
 
 #: merge-ort.c:2733 merge-recursive.c:3283
 #, c-format
 msgid ""
 "CONFLICT (file location): %s renamed to %s in %s, inside a directory that "
 "was renamed in %s, suggesting it should perhaps be moved to %s."
 msgstr ""
+"КОНФЛІКТ (розташування файлу): %s перейменовано на %s у %s всередині "
+"директорії, яку було перейменовано у %s, що свідчить про те, що його слід "
+"перемістити до %s."
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
"КОНФЛІКТ (розташування файлу): %s перейменовано на %s в %s всередині "
"директорії, яку було перейменовано в %s, що свідчить про те, що його слід "
"перемістити до %s."
```

File: po/uk.po [hunk 12]
```diff
@@ -20493,46 +20512,60 @@ msgid ""
...
 
 #: merge-ort.c:2892
 #, c-format
 msgid "CONFLICT (rename/rename): %s renamed to %s in %s and to %s in %s."
-msgstr ""
+msgstr "КОНФЛІКТ (rename/rename): %s перейменовано на %s у %s та на %s у %s."
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
msgstr "КОНФЛІКТ (перейменовано/перейменовано): %s перейменовано на %s в %s та на %s в %s."
```

File: po/uk.po [hunk 13]
```diff
@@ -20541,11 +20574,15 @@ msgid ""
 "conflicts AND collides with another path; this may result in nested conflict "
 "markers."
 msgstr ""
+"КОНФЛІКТ (перейменування призвело до зіткнення): перейменування %s -> %s має "
+"конфлікт вмісту і зіткнулося з іншим шляхом; це може призвести до появи "
+"вкладених маркерів конфлікту."
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
"КОНФЛІКТ (перейменування призвело до колізії): перейменування %s -> %s має "
"конфлікт вмісту та зіткається з іншим шляхом; це може призвести до появи "
"вкладених маркерів конфлікту."
```

File: po/uk.po [hunk 14]
```diff
@@ -20541,11 +20574,15 @@ msgid ""
...
 
 #: merge-ort.c:3010 merge-ort.c:3035
 #, c-format
 msgid "CONFLICT (rename/delete): %s renamed to %s in %s, but deleted in %s."
 msgstr ""
+"КОНФЛІКТ (rename/delete): %s перейменовано на %s у %s, але видалено у %s."
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
"КОНФЛІКТ (перейменовано/видалено): %s перейменовано на %s в %s, але видалено в %s."
```

File: po/uk.po [hunk 15]
```diff
@@ -20563,44 +20600,52 @@ msgid ""
...
+"до %s замість цього."
 
 #: merge-ort.c:4039
 #, c-format
 msgid ""
 "CONFLICT (distinct types): %s had different types on each side; renamed both "
 "of them so each can be recorded somewhere."
 msgstr ""
+"КОНФЛІКТ (різні типи): %s мали різні типи з обох сторін; перейменовано "
+"обидва типи, щоб кожен з них можна було десь записати."
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
"КОНФЛІКТ (різні типи): %s мали різні типи з обох боків; перейменовано "
"обидва, щоб кожен можна було десь записати."
```

File: po/uk.po [hunk 16]
```diff
@@ -20563,44 +20600,52 @@ msgid ""
...
 
 #: merge-ort.c:4048
 #, c-format
 msgid ""
 "CONFLICT (distinct types): %s had different types on each side; renamed one "
 "of them so each can be recorded somewhere."
 msgstr ""
+"КОНФЛІКТ (різні типи): %s мали різні типи з обох сторін; перейменовано один "
+"з них, щоб кожен можна було десь записати."
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
"КОНФЛІКТ (різні типи): %s мали різні типи з обох боків; перейменовано один "
"з них, щоб кожен можна було десь записати."
```

File: po/uk.po [hunk 17]
```diff
@@ -20563,44 +20600,52 @@ msgid ""
...
 
 #: merge-ort.c:4135 merge-recursive.c:3113
 msgid "content"
-msgstr ""
+msgstr "вміст"
 
 #: merge-ort.c:4137 merge-recursive.c:3117
 msgid "add/add"
-msgstr ""
+msgstr "додати/додати"
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
msgstr "додано/додано"
```

File: po/uk.po [hunk 18]
```diff
@@ -20563,44 +20600,52 @@ msgid ""
...
 #: merge-ort.c:4139 merge-recursive.c:3162
 msgid "submodule"
-msgstr ""
+msgstr "підмодуль"
 
 #: merge-ort.c:4142 merge-recursive.c:3163
 #, c-format
 msgid "CONFLICT (%s): Merge conflict in %s"
-msgstr ""
+msgstr "КОНФЛІКТ (%s): Конфлікт злиття у %s"
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
msgstr "КОНФЛІКТ (%s): конфлікт злиття в %s"
```

File: po/uk.po [hunk 19]
```diff
@@ -20563,44 +20600,52 @@ msgid ""
...
 
 #: merge-ort.c:4187
 #, c-format
 msgid ""
 "CONFLICT (modify/delete): %s deleted in %s and modified in %s.  Version %s "
 "of %s left in tree."
 msgstr ""
+"КОНФЛІКТ (modify/delete): %s видалено у %s та змінено у %s.  Версію %s від "
+"%s залишено у дереві."
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
"КОНФЛІКТ (змінено/видалено): %s видалено в %s та змінено в %s.  Версію %s з "
"%s залишено у дереві."
```

File: po/uk.po [hunk 20]
```diff
@@ -20614,6 +20659,8 @@ msgid ""
 " - go to submodule (%s), and either merge commit %s\n"
 "   or update to an existing commit which has merged those changes\n"
 msgstr ""
+" - перейдіть до підмодуля (%s) і або зробіть злиття коміту %s\n"
+"   або зробіть оновлення до існуючого коміту, який додав ці зміни\n"
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
" - перейдіть до підмодуля (%s), щоб або злити коміт %s\n"
"   або оновитись до існуючого коміту, в якому ці зміни вже злиті\n"
```

File: po/uk.po [hunk 21]
```diff
@@ -20629,18 +20676,28 @@ msgid ""
...
+"Рекурсивне злиття з підмодулями наразі підтримує лише тривіальні випадки.\n"
+"Будь ласка, обробляйте злиття кожного конфліктного підмодуля вручну.\n"
+"Це можна зробити за допомогою наступних кроків:\n"
+"%s - поверніться до суперпроекту і виконайте:\n"
+"\n"
+"      git add %s\n"
+"\n"
+"   щоб записати вищезгадане злиття або оновлення\n"
+"- розвʼяжіть будь-які інші конфлікти у суперпроекті\n"
+"- зробіть коміт отриманого індексу в суперпроекті\n"
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
"Рекурсивне злиття з підмодулями наразі підтримує лише тривіальні випадки.\n"
"Будь ласка, обробіть злиття кожного конфліктного підмодуля вручну.\n"
"Це можна зробити за допомогою наступних кроків:\n"
"%s - поверніться до суперпроекту і виконайте:\n"
"\n"
"      git add %s\n"
"\n"
"   щоб записати вищезгадане злиття або оновлення\n"
"- розвʼяжіть будь-які інші конфлікти в суперпроекті\n"
"- зробіть коміт підсумкового індекса в суперпроекті\n"
```

File: po/uk.po [hunk 22]
```diff
@@ -20689,31 +20748,31 @@ msgstr "не вдалося зробити символьне посилання
...
 #, c-format
 msgid "do not know what to do with %06o %s '%s'"
-msgstr ""
+msgstr "не знаю, що робити з %06o %s \"%s\""
 
 #: merge-recursive.c:1245 merge-recursive.c:1258
 #, c-format
 msgid "Fast-forwarding submodule %s to the following commit:"
-msgstr ""
+msgstr "Перемотування вперед підмодуля %s до наступного коміту:"
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
msgstr "Перемотування підмодуля %s вперед до наступного коміту:"
```

File: po/uk.po [hunk 23]
```diff
@@ -20689,31 +20748,31 @@ msgstr "не вдалося зробити символьне посилання
...
 
 #: merge-recursive.c:1248 merge-recursive.c:1261
 #, c-format
 msgid "Fast-forwarding submodule %s"
-msgstr ""
+msgstr "Перемотування вперед підмодуля %s"
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
msgstr "Перемотування підмодуля %s вперед"
```

File: po/uk.po [hunk 24]
```diff
@@ -20725,44 +20784,60 @@ msgid ""
 "\n"
 "which will accept this suggestion.\n"
 msgstr ""
+"Якщо це правильно, просто зробіть додавання до індексу, наприклад\n"
+"за допомогою:\n"
+"\n"
+"  git update-index --cacheinfo 160000 %s \"%s\"\n"
+"\n"
+"який прийме цю пропозицію.\n"
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
"Якщо все вірно, просто додайте це до індекса, наприклад,\n"
"використавши:\n"
"\n"
"  git update-index --cacheinfo 160000 %s \"%s\"\n"
"\n"
"який прийме цю пропозицію.\n"
```

File: po/uk.po [hunk 25]
```diff
@@ -20725,44 +20784,60 @@ msgid ""
...
 msgid "Failed to merge submodule %s (multiple merges found)"
-msgstr ""
+msgstr "Не вдалося злити підмодуль %s (знайдено більше одного злиття)"
 
 #: merge-recursive.c:1446
 #, c-format
 msgid "Error: Refusing to lose untracked file at %s; writing to %s instead."
 msgstr ""
+"Помилка: Відмова від втрати невідстежуваного файла за адресою %s; натомість "
+"записуємо до %s."
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
"Помилка: відмова від втрати невідстежуваного файла %s; натомість "
"записуємо до %s."
```

File: po/uk.po [hunk 26]
```diff
@@ -20725,44 +20784,60 @@ msgid ""
...
 
 #: merge-recursive.c:1518
 #, c-format
 msgid ""
 "CONFLICT (%s/delete): %s deleted in %s and %s in %s. Version %s of %s left "
 "in tree."
 msgstr ""
+"КОНФЛІКТ (%s/delete): %s видалено у %s та %s у %s. Версію %s від %s "
+"залишено у дереві."
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
"КОНФЛІКТ (%s/видалено): %s видалено в %s та %s в %s. Версію %s з %s "
"залишено в дереві."
```

File: po/uk.po [hunk 27]
```diff
@@ -20725,44 +20784,60 @@ msgid ""
...
 
 #: merge-recursive.c:1523
 #, c-format
 msgid ""
 "CONFLICT (%s/delete): %s deleted in %s and %s to %s in %s. Version %s of %s "
 "left in tree."
 msgstr ""
+"КОНФЛІКТ (%s/delete): %s видалено у %s та %s до %s у %s. Версію %s від %s "
+"залишено у дереві."
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
"КОНФЛІКТ (%s/видалено): %s видалено в %s і %s в %s в %s. Версію %s з %s "
"залишено у дереві."
```

File: po/uk.po [hunk 28]
```diff
@@ -20725,44 +20784,60 @@ msgid ""
...
 
 #: merge-recursive.c:1530
 #, c-format
 msgid ""
 "CONFLICT (%s/delete): %s deleted in %s and %s in %s. Version %s of %s left "
 "in tree at %s."
 msgstr ""
+"КОНФЛІКТ (%s/delete): %s видалено у %s та %s у %s. Версію %s від %s "
+"залишено у дереві в %s."
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
"КОНФЛІКТ (%s/видалено): %s видалено в %s і %s в %s. Версію %s з %s "
"залишено у дереві в %s."
```

File: po/uk.po [hunk 29]
```diff
@@ -20725,44 +20784,60 @@ msgid ""
...
 
 #: merge-recursive.c:1535
 #, c-format
 msgid ""
 "CONFLICT (%s/delete): %s deleted in %s and %s to %s in %s. Version %s of %s "
 "left in tree at %s."
 msgstr ""
+"КОНФЛІКТ (%s/delete): %s видалено у %s та %s до %s у %s. Версію %s від %s "
+"залишено у дереві в %s."
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
"КОНФЛІКТ (%s/видалено): %s видалено в %s і %s в %s в %s. Версію %s з %s "
"залишено в дереві в %s."
```

File: po/uk.po [hunk 30]
```diff
@@ -20781,37 +20856,45 @@ msgstr "Відмова від втрати брудного файла %s"
 #, c-format
 msgid "Refusing to lose untracked file at %s, even though it's in the way."
 msgstr ""
+"Відмова від втрати невідстежуваного файлу за адресою %s, не дивлячись на те, "
+"що він знаходиться на шляху."
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
"Відмова від втрати невідстежуваного файла %s, не дивлячись на те, "
"що він знаходиться на шляху."
```

File: po/uk.po [hunk 31]
```diff
@@ -20781,37 +20856,45 @@ msgstr "Відмова від втрати брудного файла %s"
...
 
 #: merge-recursive.c:1689
 #, c-format
 msgid "CONFLICT (rename/add): Rename %s->%s in %s.  Added %s in %s"
-msgstr ""
+msgstr "КОНФЛІКТ (rename/add): Перейменувати %s->%s у %s.  Додано %s до %s"
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
msgstr "КОНФЛІКТ (перейменовано/додано): перейменовано %s->%s в %s.  Додано %s в %s"
```

File: po/uk.po [hunk 32]
```diff
@@ -20781,37 +20856,45 @@ msgstr "Відмова від втрати брудного файла %s"
...
 
 #: merge-recursive.c:1720
 #, c-format
 msgid "%s is a directory in %s adding as %s instead"
-msgstr ""
+msgstr "%s є директорією у %s замість цього додаємо як %s"
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
msgstr "%s є директорією в %s, натомість додаємо як %s"
```

File: po/uk.po [hunk 33]
```diff
@@ -20781,37 +20856,45 @@ msgstr "Відмова від втрати брудного файла %s"
...
 
 #: merge-recursive.c:1725
 #, c-format
 msgid "Refusing to lose untracked file at %s; adding as %s instead"
 msgstr ""
+"Відмова від втрати невідстежуваного файла за адресою %s; натомість додаємо "
+"як %s"
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
"Відмова від втрати невідстежуваного файла %s; натомість додаємо "
"як %s"
```

File: po/uk.po [hunk 34]
```diff
@@ -20781,37 +20856,45 @@ msgstr "Відмова від втрати брудного файла %s"
...
 
 #: merge-recursive.c:1752
 #, c-format
 msgid ""
 "CONFLICT (rename/rename): Rename \"%s\"->\"%s\" in branch \"%s\" rename "
 "\"%s\"->\"%s\" in \"%s\"%s"
 msgstr ""
+"КОНФЛІКТ (rename/rename): Перейменувати \"%s\"->\"%s\" у гілці \"%s\" "
+"перейменувати \"%s\"->\"%s\" у \"%s\"%s"
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
"КОНФЛІКТ (перейменовано/перейменовано): перейменовано \"%s\"->\"%s\" у гілці \"%s\" "
"перейменовано \"%s\"->\"%s\" у \"%s\"%s"
```

File: po/uk.po [hunk 35]
```diff
@@ -20781,37 +20856,45 @@ msgstr "Відмова від втрати брудного файла %s"
...
 msgid " (left unresolved)"
-msgstr ""
+msgstr "(залишилося нерозвʼязаним)"
 
 #: merge-recursive.c:1849
 #, c-format
 msgid "CONFLICT (rename/rename): Rename %s->%s in %s. Rename %s->%s in %s"
 msgstr ""
+"КОНФЛІКТ (rename/rename): Перейменувати %s->%s у %s. Перейменувати %s->%s у "
+"%s"
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
"КОНФЛІКТ (перейменовано/перейменовано): перейменовано %s->%s в %s. Перейменовано %s->%s в "
"%s"
```

File: po/uk.po [hunk 36]
```diff
@@ -20820,13 +20903,18 @@ msgid ""
 "directory %s was renamed to multiple other directories, with no destination "
 "getting a majority of the files."
 msgstr ""
+"КОНФЛІКТ (розділення під час перейменування директорії): Неясно, куди "
+"помістити %s, оскільки директорію %s було перейменовано на декілька інших "
+"директорій, при цьому більшість файлів не потрапило до жодної з них."
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
"КОНФЛІКТ (розбіжність під час перейменування директорії): Неясно, куди "
"помістити %s, оскільки директорію %s було перейменовано в декілька інших "
"директорій,  жодна з яких не отримала більшості файлів."
```

File: po/uk.po [hunk 37]
```diff
@@ -20820,13 +20903,18 @@ msgid ""
...
 
 #: merge-recursive.c:2246
 #, c-format
 msgid ""
 "CONFLICT (rename/rename): Rename directory %s->%s in %s. Rename directory %s-"
 ">%s in %s"
 msgstr ""
+"КОНФЛІКТ (rename/rename): Перейменувати директорію %s->%s у %s. "
+"Перейменувати директорію %s->%s у %s"
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
"КОНФЛІКТ (перейменовано/перейменовано): перейменовано директорію %s->%s в %s. "
"Перейменовано директорію %s->%s в %s"
```

File: po/uk.po [hunk 38]
```diff
@@ -20839,12 +20927,12 @@ msgstr "змінено"
 #: merge-recursive.c:3140
 #, c-format
 msgid "Skipped %s (merged same as existing)"
-msgstr ""
+msgstr "Пропущено %s (злиття таке ж як існуюче)"
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
msgstr "Пропущено %s (злите - те саме, що й існуюче)"
```

File: po/uk.po [hunk 39]
```diff
@@ -20857,43 +20945,43 @@ msgstr "файл/директорія"
...
 #: merge-recursive.c:3425
 msgid "directory/file"
-msgstr ""
+msgstr "директорія/файл"
 
 #: merge-recursive.c:3432
 #, c-format
 msgid "CONFLICT (%s): There is a directory with name %s in %s. Adding %s as %s"
-msgstr ""
+msgstr "КОНФЛІКТ (%s): У %s існує директорія з іменем %s. Додавання %s як %s"
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
msgstr "КОНФЛІКТ (%s): в %s існує директорія з іменем %s. Додавання %s як %s"
```

File: po/uk.po [hunk 40]
```diff
@@ -20857,43 +20945,43 @@ msgstr "файл/директорія"
...
 #, c-format
 msgid "Adding %s"
-msgstr ""
+msgstr "Додавання %s"
 
 #: merge-recursive.c:3450
 #, c-format
 msgid "CONFLICT (add/add): Merge conflict in %s"
-msgstr ""
+msgstr "КОНФЛІКТ (add/add): Конфлікт злиття у %s"
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
msgstr "КОНФЛІКТ (додано/додано): Конфлікт злиття у %s"
```

File: po/uk.po [hunk 41]
```diff
@@ -21078,7 +21168,9 @@ msgstr "Перевірка OID послідовності в multi-pack-index"
 #: midx.c:1772
 #, c-format
 msgid "oid lookup out of order: oid[%d] = %s >= %s = oid[%d]"
-msgstr "пошук oid у неправильній послідовності: oid[%d] = %s >= %s = oid[%d]"
+msgstr ""
+"oid lookup знаходиться у неправильній послідовності: oid[%d] = %s >= %s = "
+"oid[%d]"
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
"невірна послідовність oid lookup: oid[%d] = %s >= %s = "
"oid[%d]"
```

**Review by arkid15r — COMMENTED** [review]
This was a tough one

👍

File: po/uk.po (hunk 3 above)
Comment (kasya) [reply po/uk.po]: 👏🏼

**Pull request merged**